dev_tools/nbfmt

```diff
@@ -18,23 +18,34 @@
 cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" || exit
 cd "$(git rev-parse --show-toplevel)" || exit
 
-# Check if cirq/check/nbformat exists, if not grab it.
+# Base URL for downloading tools from Cirq.
+declare -r cirq=https://raw.githubusercontent.com/quantumlib/Cirq/main
```

Comment:
Are we planning to download from cirq from other repos too?

Reply:
> Are we planning to download from cirq from other repos too?

Ah, no; the reason I assigned a variable is that the previous code used the same long URL in two places. Seemed better to keep things DRY and write it once.

Reply:
Ok, maybe I don't understand your question :-). 

You had asked "Are we planning to download from cirq from other repos too?", which I interpreted as asking whether we'd download cirq from repos other than the cirq one. The answer to that is no – I can't think of a reason to download it from a repo _other_ than cirq's repo. I was puzzled by why you were asking, so I thought maybe the reason you were asking is that it's unclear why the download location is being assigned to a variable when that value will never change. The reason for _that_ is that the location is used in more than one place in the script.

Could you clarify what "download from cirq from other repos" means?

Reply:
This PR is for the unitary repo but the url is downloading tools from the cirq repo:

```# Base URL for downloading tools from Cirq.
declare -r cirq=https://raw.githubusercontent.com/quantumlib/Cirq/main```

I think we should keep the repos self-contained and not download tools from the cirq repo unless there is a good reason.

Reply:
Oh, I understand now.

The reason for this is simply that the original `nbfmt` in unitary did this. (This can be seen in [the version in main currently](https://github.com/quantumlib/unitary/blob/main/dev_tools/nbfmt).) I didn't want to question the original choice so left that part the same, and only cleaned up the code in other ways.

I could rewrite that, if that would be preferable?